content/models/social_force_model/index.md

```diff
@@ -0,0 +1,118 @@
+
+## Obstacle force
+Obstacles exert a force on the agent similar to a static agent.
+The repulsive force is increases exponentially with decreasing distance. When the Obstacle collides with the agent the pushing forces are increased additionally. When colliding, a frictional force also occurs, which acts orthogonally to the repulsive force in the direction of the speed of the agent.
+
+$$ \overrightarrow{F\_i^{\mathrm{obst}}} = \sum_{W} \overrightarrow{f_{iW}}$$
+
+$$\overrightarrow{f_{iW}} = [A_i exp[(r_{i} - d_{iW})/B_i] + kg(r_{i} - d_{iW})] \overrightarrow{n_{iW}} + \kappa g(r_{i} - d_{iW}) (\overrightarrow{v\_{i}}\cdot\overrightarrow{t_{iW}}) \overrightarrow{t_{iW}}$$
+
+$$\phantom{.........}\fbox{\phantom{.....................} pushing \phantom{...................}} \fbox{\phantom{...........} sliding \phantom{...........}}$$
+
+* $W$ is a wall segment
+* $r_i$ is the radius agent $i$
+* $d_{iW}$ is distance from closest point on the wall segement to $i$
+* $\overrightarrow{n_{iW}} = (n\_{iW}^1, n\_{iW}^2) = (\overrightarrow{pos\_i} - \overrightarrow{pos\_W})/d\_{iW}$ the direction from $\overrightarrow{pos\_W}$ the closest point on the wall  to $i$
+* $ \overrightarrow{t_{iW}} = (- n\_{iW}^2, n\_{iW}^1)$ is the tangent of $\overrightarrow{n_{iW}}$ which is perpendicular to it, rotated counterclockwise
+* $v\_i$ is the velocity of $i$
+* $A_i$, $B_i$, $k$ and $\kappa$ are constants 
+  -  $A_i$ obstacle scale
+  -  $B_i$ force distance
+  -  $k$ body force
+  -  $\kappa$ friction
+* $g$ is a function with $g(x) =  \begin{cases}
+                                      x \text{ if } x < 0 \\
+                                      \text{ else } 0
+                                  \end{cases}$
+
+
+{{< figure src="Pushing_Obstacle_Force.png" caption="Direction of repulsive forces acting on an agent" width="650">}}
+
+the repulsive force originating from a wall segment acts from the point on the wall segment that is closest to the agent. This point is marked in magenta in the illustration above.
+
+{{< figure src="Sliding_Obstacle_Force.png" caption="Direction of the pushing and frictional forces acting on agents colliding with a wall"  width="800">}}
+
+If the minimal distance to a wallsegment is smaller than the radius of an agent an additional frictional force occurs. This frictional force acts orthogonally to the repulsive force in the direction of the speed difference of the two agents.
+
+## Calculating new velocity and new position from forces
+With the definition of the forces affecting an individual, it is possible to calculate its new speed and position.
+
+$$ \overrightarrow{v\_{new}} = \overrightarrow{v\_i} + [\overrightarrow{F\_i^{\mathrm{drv}}} + \frac{\overrightarrow{F\_i^{\mathrm{ped}}} + \overrightarrow{F\_i^{\mathrm{obst}}}}{m\_i}] \cdot \delta T$$
```

Comment:
obs or wall?
Maybe use "O" for obstacle and mention that obstacles are made of lines, then your figures still make sense.

Reply:
new and pos are not variables, so should not be cursive

Reply:
Equations need punctuation as well.